docs/release/trg-0/trg-1-4.md

```diff
@@ -16,7 +16,7 @@ Also updating the static files will be easier and faster to do for every committ
 
 ## Description
 
-All diagram files **must** be created with code and either stored as `.svg` file within our repositories or implemented in your documentation.
+All diagram files **must** be either created with code languages or stored as `.svg` file within our repositories. If your using Draw.io please also describe how and with which version you created the used diagram in your `README.md` documentation file.
```

Comment:
"code languages" -> maybe diagram as code tool? never heard the term "code languages" before
languages, or -> comma missing
If `you're` instead of `your`.
What is completely missing from my point of view is a specific example on how to document in the `README.`
Maybe this is not as relevant for product repos as it will be for the website. But especially for the website, we have a lot of different contributors and very deep directory structure. If we clutter our top level `README` with various instruction on how to handle a file deep into our directory structure, that will potentially mess up our entrypoint to the repo. 
Have there been thoughts on how to handle this in our website repo already?

Reply:
fixed typos and update the current one to just document it, not exactly where. 

[chore: restructure content + update example diagram](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/429/commits/f3adad9638f7fbfec8c422bd4f82d6ddc0edb740)